tools/test-generator/templates/ocaml/forth/test.ml

```diff
@@ -109,6 +117,7 @@ let () =
         drop_tests;
         swap_tests; 
         over_tests; 
-        user_defined_words_tests
+        user_defined_words_tests;
+        case_insensitivity
```

Comment:
Terminate each element in a list literal with `;` to avoid this kind of diff behavior.
```suggestion
        case_insensitivity;
```